### ISSUE TITLE
Feature request GTK 3; don't use symbolic icons in toolbars either

### ISSUE THREAD

**Issue by hhhorb** [shimmerproject/Greybird#83]
It's fixed in sidebars (gtk-wisgets.css) with -gtk-icon-style: regular; is there selector for toolbar buttons too, like when you acces the "Open file" menu item in file-roller, transmission-gtk, etc?

ps: I love your themes, I've used them for years now. Cheers.

**Comment by ochosi** [comment]
Hi!
Thanks – always nice to hear :)
What distribution or version of Gtk3 are you using? I have colorful toolbar icons here in transmission (but could be an Ubuntu patch, on 15.04).

![screenshot_2015-04-09_10-52-15](https://cloud.githubusercontent.com/assets/261436/7063383/80c8a79e-dea6-11e4-9630-568d81eaba76.png)

**Comment by hhhorb** [comment]
hey, thanks for the response! Sorry for the lack of details. I'm using Debian jessie, so GTK 3.14. The main Transmission interface is fine, but go to File>Open (I've modified the theme, but it looks similar in Greybird-master from this Git)...
<a href="http://imgur.com/sMY6Kcz"><img src="http://i.imgur.com/sMY6Kcz.png" title="source: imgur.com" /></a>

**Comment by ochosi** [comment]
Right, so you're referring to the inline toolbar up there. Unfortunately I haven't found a way to fix that yet, even gtkinspector is leaving me out in the cold with that one...

**Comment by hhhorb** [comment]
OK, no problem. If I find out anything, I'll post.

Cheers!

**Comment by hhhorb** [comment]
Well, after a couple of hours of research, the only thing of real value I've found is this email...
https://mail.gnome.org/archives/gtk-devel-list/2014-May/msg00020.html

The relevant bits... "4. Forcing icon styles: The -gtk-icon-style property... For elements that render icons from the icon theme, such as GtkImage, GtkEntry or GtkCellRendererPixbuf, this property allows to override the style to be used."

And... "1. The -gtk-icontheme image... ISSUE 2
For now, the -gtk-icon-style property does not influence the behavior of -gtk-icontheme(). You need to specify symbolicness of the icon yourself by using the right icon name."

So I'm guessing that either those file-chooser toolbar's css either specify a symbolic icon or aren't rendered by a property that allows it to be overridden... or something. :^D

**Comment by ochosi** [comment]
Thanks for looking into this. Another good way to look for fixes would be finding a theme which successfully does that. But yeah, maybe it isn't possible.
